langchain/package.json

```diff
@@ -637,7 +640,7 @@
     "@xata.io/client": "^0.25.1",
     "@zilliz/milvus2-sdk-node": ">=2.2.11",
     "apify-client": "^2.7.1",
-    "axios": "^0.26.0",
+    "axios": "^0.26.1",
```

Comment:
Can we do this with just `fetch`? Otherwise people will need to install packages to use this

Reply:
Which is better between using axios@0.26.0 or fetch?
Because axios@0.26.0 is also working for me.

Reply:
I resolved with fetch